inc/class-core-sitemaps-users.php

```diff
@@ -1,7 +1,6 @@
 <?php
 /**
  * The Core_Sitemaps_Users sitemap provider.
- *
```

Comment:
This blank line should stay. Usually the first line is a Description and following lines, separated by a blank line, are a longer description. Think of it like an email subject + body.